pkg/apis/integreatly/v1alpha1/grafana_types.go

```diff
@@ -465,6 +466,7 @@ type GrafanaConfigPlugins struct {
 // +k8s:openapi-gen=true
 type GrafanaStatus struct {
 	Phase               StatusPhase                       `json:"phase"`
+	PreviousServiceName string                            `json:"previousServiceName"`
```

Comment:
Storing this var in the status field allows for persistence between separate runs of the operator. using a local var we would lose the reference.